main.cs

```diff
@@ -40,5 +54,13 @@ public override void OnGameStart(bool storyMode)
             }
             LyokoLogger.Log(Name,"Game start event detected.");
         }
+
+
+
+
+        public static void Log(string message)
+        {
+            LyokoLogger.Log("TenSuperscan", message);
+        }
```

Comment:
Why not use this for other Logging on Main?

Reply:
Didn't bother to refractor because there's no benefit in doing so